scripts/gha/integration_testing/gameloop_android/gradle.properties

```diff
@@ -0,0 +1,21 @@
+# Project-wide Gradle settings.
+# IDE (e.g. Android Studio) users:
+# Gradle settings configured through the IDE *will override*
+# any settings specified in this file.
+# For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+org.gradle.jvmargs=-Xmx2048m
+# When configured, Gradle will run in incubating parallel mode.
+# This option should only be used with decoupled projects. More details, visit
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+# AndroidX package structure to make it clearer which packages are bundled with the
+# Android operating system, and which are packaged with your app"s APK
+# https://developer.android.com/topic/libraries/support-library/androidx-rn
+android.useAndroidX=true
```

Comment:
Does this mean that these tests can only run on devices which have atleast the minimum supported version for androidx and jetifier?

Reply:
androidx and jetifier are not the problem here.
uiautomator requires minimum version 18.

Reply:
Good to know. Maybe mention this somewhere in code comments if it hasn't been already?

Reply:
I think that's fine for our purposes - we'll still want to use FTL for a range of old devices and stuff. This is just so we can quickly run tests on Android without using FTL.